ocs_ci/ocs/machine.py

```diff
@@ -168,6 +227,31 @@ def add_node(machine_set, count):
     return True
 
 
+def wait_for_new_node_to_be_ready(machine_set):
+    """
+     Wait for the new node to be in ready state
+
+     Args:
+         machine_set (str): Name of the machine set
+
+    Raises:
+        CommandFailed: In case the new spun machine fails to come
+     """
+    count = 0
+    replica_count = get_replica_count(machine_set)
+    try:
+        time.sleep(300)
```

Comment:
please use TimeoutingSampler.

You can start his function and the machine will be in running state and waiting 300s is just a waste of time

Reply:
Done.